aw-server/Cargo.toml

```diff
@@ -35,10 +35,10 @@ aw-models = { path = "../aw-models" }
 aw-transform = { path = "../aw-transform" }
 aw-query = { path = "../aw-query" }
 
-[target.'cfg(target_os="linux")'.dependencies]
+[target.'cfg(all(target_os="linux", target_arch="x86"))'.dependencies]
```

Comment:
@johan-bjareholt I assume this is what you intended?

Reply:
Correct!